starter-code/src/App.js

```diff
@@ -1,25 +1,36 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import CountryList from "./components/CountryList";
+import CountryDetail from "./components/CountryDetail";
+import { Switch, Route } from "react-router-dom";
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        <div>
+          <nav class="navbar navbar-dark bg-primary mb-3">
+            <div class="container">
+              <a class="navbar-brand" href="/">
```

Comment:
You should have used `Link` here rather than an a tag because we don't want the page to refresh when you click on the link